src/components/stepforms/widgets/DateComponents/DateRangeInput.vue

```diff
@@ -553,4 +553,11 @@ export default class DateRangeInput extends Vue {
 .widget-date-input__editor--compact {
   max-width: calc(100% - 20px);
 }
+
+.widget-date-input--compact.widget-date-input--reset {
+  // hide calendar icon in compact mode to have more space to display label
+  .widget-date-input__type-icon {
+    display: none;
+  }
+}
```

Comment:
Maybe this would be even more explicit with  `v-if="compactMode && !!value"` directly on the element?

Reply:
Why not, i can yes

Reply:
Finally is not really easier to ready for my own opinion and as the classes are used for other cases, we won't clean any other code, so i keeped it this way